parsing/dialects/dialect1/src/test/java/org/apache/calcite/test/Dialect1ParserTest.java

```diff
@@ -2674,4 +2674,72 @@ private void checkExNotNull(SqlParserUtil.StringAndPos sap, Throwable thrown) {
     final String expected = "(?s).*Encountered \"\\( -\" at .*";
     sql(sql).fails(expected);
   }
+
+  @Test public void testPeriodTypeNameSpecsDate() {
```

Comment:
nit: Extra "s" after Spec (Here and also rest of the test cases)

Reply:
Changed in https://github.com/googleinterns/calcite/pull/177/commits/110d83e563426a9ea05113263c6efc02c44447dc